src/Uno.UI.Wasm/ts/WindowManager.ts

```diff
@@ -372,7 +372,16 @@
 
 			for (const name in properties) {
 				if (properties.hasOwnProperty(name)) {
-					(element as any)[name] = properties[name];
+					var setVal = properties[name];
+					if (setVal === "true") {
+						(element as any)[name] = true;
```

Comment:
What happen if we try to set the string value `"true"` here?
+ maybe we should handle `null` the same way too, no?

Reply:
* Setting `input["disabled"]` to any string (including `"false"`) is equivalent to setting it to `true`, ie the `TextBoxView` stays disabled.
 * There may be other cases where we want to convert strings to literals, yes. Great candidate for a separate PR. ;)